API-strategie-modulen/API-strategie-mod-geo/crs.md

```diff
@@ -138,15 +138,16 @@ The CRS can be specified for request and response individually using parameters
 In an API that supports the creation of items, POST requests with geospatial content in the body may be sent by a client to the server. In that case, it is necessary to indicate the CRS used, unless CRS84, the default CRS, is used.
 
 <div class="rule" id="api-geo-11">
-  <p class="rulelab"><strong>API-GEO-11</strong>: When HTTP POST requests are supported, pass the coordinate reference system (CRS) of geometry in the request body as a header</p>
+  <p class="rulelab"><strong>API-GEO-11</strong>: When HTTP POST, PUT and/or PATCH requests are supported, pass the coordinate reference system (CRS) of geometry in the request body as a header</p>
   <p>Support the <a href="http://docs.ogc.org/DRAFTS/20-002.html#feature-crs">OGC API Features part 4 <code>Content-Crs</code> header</a> in conformance to the standard.</p>
   <p>Alternatively, if the feature representation supports expressing CRS information for each feature / geometry, the information can also be included in the feature representation. If no CRS is asserted, the default CRS, CRS84, is assumed, as stated in <a href="#api-geo-7">API-GEO-7</a>.<p>
   <h4 class="rulelab">How to test</h4>
-  <p>In a request (i.e. when creating an item on the server):</p>
+  <p>In a request (i.e. when creating or updating an item on the server):</p>
```

Comment:
Ook DELETE?

Reply:
Het leek mij niet nodig om DELETE hier te noemen, omdat je daarbij geen coördinaten in de request zou sturen en dus ook geen CRS hoeft aan te geven.